make/harbor.yml.tmpl

```diff
@@ -246,7 +246,16 @@ upload_purging:
   interval: 24h
   dryrun: false
 
-# Cache related config
+# cache layer configurations
+# if this feature enabled, harbor will cache the resource
+# `project/project_metadata/repository/artifact/manifest` in the redis
+# which especially help to improve the performance of high concurrent pulling manifest.
```

Comment:
`which can especially help to improve the performance of high concurrent manifest pulling.`
might be better

Reply:
updated, thanks~